index.html

```diff
@@ -1927,79 +908,48 @@ <h4>Error Responses</h4>
       </section>
     </section>
 
-    <!-- External Representation -->
-    <section id="external-representation">
-      <h2 id="external-td-representations">External TD
-        representations</h2>
-      <p>
-        <span class="rfc2119-assertion" id="profile-5.3-exteanal-td-representations-1">
-        The default representation is JSON. Semantic
-        annotations based on JSON-LD MAY be present but are not
-        required to perform all interactions with the thing
-        instance.
-        </span>
+    <!-- Links -->
+    <section id="links">
+      <h3>Links</h3>
+      <p class="ednote">
+        This section will define link relation types which conformant Web Things
+        MAY use and conformant Consumers MAY support
+        (e.g. <code>{ "rel": "alternate", "type": "text/html"}</code> to link
+        to an HTML user interface).
       </p>
+    </section>
 
-        <section id="canonical-representation">
-          <h2 id="canonical-td-representations">Canonical TD
-            representation</h2>
-          <p>
-            A canonical representation serves multiple purposes.
-            It is simplifying the parsing process, enables to identify
-            equivalent TDs by simple string comparisons.
-            Furthermore it allows the use of a simple signing mechanism,
-            such as <a href="https://w3c-ccg.github.io/ld-proofs/">Linked Data Proofs</a> or JSON Web Signatures [[RFC7515]] and enables
-            identity checks on encrypted TDs.
-          </p>
-	  <p>
-	     The canonical JSON representation format of a TD adopts the JSON Canonicalization Scheme (JCS) defined by
-	     <a href="https://tools.ietf.org/html/rfc8785>RFC-8785</a> including errata <a href="https://www.rfc-editor.org/errata/eid6292">6292</a>.
-	  </p>
-      </section>
-
-
+    <!-- Security -->
+    <section id="security">
```

Comment:
We need a security section, however need to discuss the supported mechanisms under implementation constraints for plugfest members.

I suggest to exclude "no sec" and require basic auth as the minimum. 
The other mechanisms need input from plugfest members.

Reply:
I agree this requires some discussion, but I strongly object to excluding the `nosec` security scheme. This is currently used by all [16 web thing libraries](https://webthings.io/framework/) in the WebThings Framework and there can be good reasons for a web thing not providing its own authentication.